docs/java-getting-started.md

```diff
@@ -258,11 +373,18 @@ If you aren't using Spring in your app, you can accomplish this with the followi
 </plugin>
 ```
 
-Once dependencies are installed, you can run your app locally.
+After installing dependencies, you can run your app locally. However, the `/db` route doesn't work as you must connect to a database. If you want to access the `/db` route when running locally, create a local Postgres database and update your local `.env` file. `heroku local`, the command used to run apps locally, automatically sets up your environment based on the `.env` file in your app's root directory. Set the `JDBC_DATABASE_URL` environment variable with your local Postgres database's connection string:
+
+```
+JDBC_DATABASE_URL=jdbc:postgresql://localhost:5432/java_database_name
+```
+Your local environment is now ready to run your app and connect to the database.
 
-<h2 data-next-message="I can run my app locally">Run the app locally</h2>
+<h2 data-next-message="I can run my app locally">Run the App Locally</h2>
 
-Start your application locally with the `heroku local` CLI command (make sure you've already run `mvn clean install`):
+Ensure you've already run `mvn clean install` before running your app locally.
+
+Start your application locally with the [`heroku local` CLI command](heroku-local):
```

Comment:
not sure if we want `heroku local` command in blue/link - 
might be best to have this in a separate line down, i.e. tell the users for more CLI command documentation click here

Reply:
Updated!